dttools/src/jx_parse.c

```diff
@@ -367,8 +371,12 @@ static jx_token_t jx_scan( struct jx_parser *s )
 					return JX_TOKEN_FALSE;
 				} else if(!strcmp(s->token,"or")) {
 					return JX_TOKEN_OR;
+				} else if (!strcmp(s->token,"||")) {
```

Comment:
This is kind of strange to group `||` and `&&` with the keyword tokens.
Please give these their own cases as `<=` and `>=` above.